shared/constants/platform.desktop.js

```diff
@@ -78,56 +78,67 @@ const darwinCacheRoot = `${getenv('HOME', '')}/Library/Caches/${envedPathOSX[run
 const darwinSandboxCacheRoot = `${getenv('HOME', '')}/Library/Group Containers/keybase/Library/Caches/${envedPathOSX[runMode]}/`
 const darwinSandboxSocketPath = path.join(darwinSandboxCacheRoot, socketName)
 
-function findSocketDialPath(): Array<string> {
-  const paths = {
-    darwin: darwinSandboxSocketPath,
-    linux: linuxSocketDialPath(),
```

Comment:
this pattern executes this code even if we're not on this platform, lets not do that